---
title: "&#39;Exit Try&#39; can only appear inside a &#39;Try&#39; statement"
ms.custom: na
ms.date: "09/22/2016"
ms.prod: "visual-studio-dev14"
ms.reviewer: na
ms.suite: na
ms.technology: 
  - "devlang-csharp"
ms.tgt_pltfrm: na
ms.topic: "article"
f1_keywords: 
  - "vbc30393"
  - "bc30393"
helpviewer_keywords: 
  - "BC30393"
ms.assetid: b8651df3-a32f-478c-a6d8-aa0ef584155f
caps.latest.revision: 11
translation.priority.ht: 
  - "de-de"
  - "ja-jp"
---
# &#39;Exit Try&#39; can only appear inside a &#39;Try&#39; statement
<CodeContentPlaceHolder>0\</CodeContentPlaceHolder> can only appear within a <CodeContentPlaceHolder>1\</CodeContentPlaceHolder> block statement. Either you have a redundant <CodeContentPlaceHolder>2\</CodeContentPlaceHolder> statement, or your <CodeContentPlaceHolder>3\</CodeContentPlaceHolder> statement appears outside the bounds of its corresponding <CodeContentPlaceHolder>4\</CodeContentPlaceHolder> block.  
  
 **Error ID:** BC30393  
  
### To correct this error  
  
1.  Locate and remove the unnecessary <CodeContentPlaceHolder>5\</CodeContentPlaceHolder> statement.  
  
2.  Move the <CodeContentPlaceHolder>6\</CodeContentPlaceHolder>statement to the appropriate place within your code.  
  
## See Also  
 [Try...Catch...Finally Statement](../vs140/try...catch...finally-statement--visual-basic-.md)   
 [Structured Exception Handling Overview for Visual Basic](assetId:///bb81af80-a735-4873-9711-6151a48e418a)